### PR TITLE
Add PGSQL to AzurePGSQL swagger changes

### PR DESCRIPTION
**Pull request by ash1625**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Tasks.json [hunk 1]
```diff
@@ -635,7 +635,7 @@
       ]
     },
     "MigratePostgreSqlAzureDbForPostgreSqlSyncTaskProperties": {
-      "x-ms-discriminator-value": "Migrate.PostgreSql.AzureDbForPostgreSql.Sync",
+      "x-ms-discriminator-value": "Migrate.PostgreSql.AzureDbForPostgreSql.SyncV2",
```
Comment (raych1) [inline specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Tasks.json]: Will this child type change cause breaking change?

**Comment by raych1** [comment]
@hitenjava , please review on the swagger spec change.  @ash1625 , can you resolve the comment and fix the spell check failures?

File: specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Tasks.json (hunk 1 above)
Comment (hitenjava) [reply specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Tasks.json]: We are making the back-end backward compatible. Same inputs and outputs will continue to work with the 'v2' type task.

**Review by hitenjava — APPROVED** [review]
Approved, but this should be merge once we are close to releasing the changes.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by temandr** [comment]
@raych1 Do you know why the sdk-automation failed to build the python sdk? There doesn't seem to be a PR made and the attached sdk doesn't correspond to the change.

**Comment by temandr** [comment]
@raych1 this is ready to be merged.

**Comment by raych1** [comment]
@temandr , can you please follow https://aka.ms/AA6h31t to fix the `PrettierCheck` issue?

**Review by raych1 — APPROVED** [review]

**Comment by temandr** [comment]
It looks like I'm not authorized to merge this request. Can someone with the needed permissions merge this change?

**Comment by majastrz** [comment]
@raych1 should have permissions to merge the PR. ARM API reviewers unfortunately don't have those permissions.

**Pull request merged**